src/App.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import {Routes , Route} from "react-router-dom"; 
 import Mockman from "mockman-js";
-import Homepage from './pages/Homepage'
-import WishList from "./pages/Wishlist";
-import Cart from "./pages/Cart";
-import Products from "./pages/Products";
-import SignUp from "./pages/auth/SignUp"
-import Login from "./pages/auth/Login"
-import Navbar from './pages/components/Navbar';
+import Homepage from './pages/home/Homepage'
+import WishList from "./pages/wishlist/Wishlist";
+import Cart from "./pages/cart/Cart";
+import Products from "./pages/products/Products";
+import SignUp from "./pages/auth/signup/SignUp"
+import Login from "./pages/auth/login/Login"
+import Navbar from './pages/components/navbar/Navbar';
```

Comment:
In your pages folder make one file and name it as index.js and export all the components over there and then import all the components in your App.js from that index file. This will reduce your lines of code too and was suggested by Tanay too

Reply:
Understood , will try this.